5calls/app/src/main/java/org/a5calls/android/a5calls/adapter/OutcomeAdapter.java

```diff
@@ -9,10 +9,16 @@
 import org.a5calls.android.a5calls.R;
 import org.a5calls.android.a5calls.model.Outcome;
 
+import java.util.Arrays;
 import java.util.List;
 
 public class OutcomeAdapter extends RecyclerView.Adapter<OutcomeAdapter.ViewHolder> {
 
+    public static final List<Outcome> DEFAULT_OUTCOMES = Arrays.asList(
+            new Outcome("unavailable", Outcome.Status.UNAVAILABLE),
```

Comment:
Use strings from strings.xml to ensure translations or other text changes come through.